2021-preseason/src/main/java/frc/robot/Robot.java

```diff
@@ -85,12 +99,48 @@ public void autonomousPeriodic() {
     }
   }
 
+  
+
   /**
    * This function is called periodically during operator control.
    */
   @Override
   public void teleopPeriodic() {
     drive.updatePower();
+    if(controller.getStartButton() || controller.getAButton())
+    {
+      if(controller.getStartButton())
+      {
+        climb.elevatorUp();
+      }
+      else
+      {
+        climb.elevatorDown();
+      }
+    }
+
+    else
+    {
+      climb.elevatorStop();
+    }
```

Comment:
I think this would be easier to read if you used `if else` instead of nesting if statements. The same below

```suggestion
    if(controller.getStartButton())
    {
      climb.elevatorUp();
    }
    else if (controller.getAButton())
    {
      climb.elevatorDown();
    }
    else
    {
      climb.elevatorStop();
    }
```